e2e/tests/gatekeeperAdministersCaseAfterGatekeeping_test.js

```diff
@@ -123,7 +123,7 @@ Scenario('Gatekeeper drafts standard directions', async (I, caseViewPage, draftS
   caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
   I.seeInTab(['Standard directions order', 'File'], 'draft-standard-directions-order.pdf');
   I.seeInTab(['Standard directions order', 'Date of issue'], dateFormat(today, 'd mmmm yyyy'));
-});
+}).retry(1); // Send letter is async for the hearing details event, if things are running slow then a data altered outside of transaction error may be raised at the end of this scenario.
```

Comment:
is this easy to read maybe?
`Scenario('name....', { retries: 1 }, (I) => {});`

Reply:
I was indecisive between the 2 and went with what I have in the end because it looks safer in my opinion. Using an objects method rather than what looks to be directly injecting a property